### DISCUSSION TITLE
More about "me"

### DISCUSSION THREAD

**Discussion by zhyiyu** [community/community#13475] [Other Feature Feedback, Questions, & Ideas]
Hi, the new "For you" tab is great, however, instead of what others do, I care more about what I did in the last few days, commits, issues, pull requests, or what I care (like the repos that I worked on in the last few weeks). Although such information are available at my homepage, navigating to my home page requires two more clicks (which is not ideal).

Please consider this suggestion and add another tab for "me".

Thank you.

**Comment by jfullstackdev** [comment]
one trick I'm doing if I'm really focused in these things is to actually create an Org, because it has the separate dashboard solely for your projects, even if you don't have members, because it's designed for collaboration that every activity should be seen and be staying for several weeks, as a guide for all
